### ISSUE TITLE
Missing countries

### ISSUE THREAD

**Issue by sheinin** [coordinates2country/coordinates2country-android#4]
I'm having difficulty identifying these countries using this library, and they are not listed in the ./data dir.

Is there a way to update the lib with these entries? Thank you.

```
anguilla
armenia
azerbaijan
bahamas
belize
bermuda
bosnia and herzegovina
bouvet island
china
christmas island
congo
cote d'ivoire
estonia
faroe islands
french guiana
gambia
georgia
gibraltar
greenland
guadeloupe
guam
hong kong china
iceland
jersey
kuwait
lithuania
luxembourg
macau china
martinique
mayotte
moldova
monaco
montserrat
netherlands antilles
new caledonia
north macedonia
northern mariana islands
palestine
panama
puerto rico
reunion
saint pierre and miquelon
san marino
sao tome and principe
serbia
slovenia
spain
suriname
swaziland
turks and caicos islands
ukraine
us virgin islands

```

**Comment by nicolas-raoul** [comment]
I reproduced the issue, thanks for letting me know!
I will fix it as soon as possible.

**Comment by nicolas-raoul** [comment]
I just released 1.8, would you mind trying it? It may take a few hours before becoming available on Maven repositories.

I added the biggest countries and sovereign states, but left the ones below. Feel free to send a pull request to add them too, similar to the `countries.csv` and `countries.xcf` changes of this commit: https://github.com/coordinates2country/coordinates2country/commit/cef4dce5b3c161cbf9aa089d28ff737c11bc48ea

```
monaco
saint pierre and miquelon
bermuda
christmas island
faroe islands
french guiana
gibraltar
guadeloupe
guam
jersey
martinique
mayotte
montserrat
netherlands antilles
northern mariana islands
reunion
san marino
turks and caicos islands
us virgin islands
bouvet island
```

**Comment by sheinin** [comment]
It's working great. Thank you!
